scripts/run-example.sh

```diff
@@ -87,7 +96,14 @@ then
   ${HELM} upgrade --install vault hashicorp/vault --namespace vault --values "${SAMPLE_DIR}/helm-vault-values.yaml" --wait
   ${KUBECTL} exec vault-0 -n vault -- vault secrets enable transit || true
   echo -e "${GREEN}HashiCorp Vault installed.${NOCOLOR}"
-  VAULT_UI_URL=$(${MINIKUBE} service vault -n vault --url 2>/dev/null | head -1)
+  pipename="${TMPDIR}/vault_url"
+  mkfifo "${pipename}"
+  ${MINIKUBE} service vault -n vault --url 2>/dev/null > "${pipename}" &
+  MINIKUBE_VAULT_PORT_FORWARD_PID=$!
+  VAULT_UI_URL=$(head -1 "${pipename}")
+  rm "${pipename}"
+  ON_SHUTDOWN+=("kill -3 ${MINIKUBE_VAULT_PORT_FORWARD_PID}")
```

Comment:
On platforms with the Apple chipset, don't you need to leave `minikube service vault` running in order to allow users to hit the Vault Console.  Shutting it down when the script ends is too soon.

However, the ability to use the Vault Console is really there as a convenience to users.  It is not actually part of instructions (we tell the user to run the vault CLI on-cluster).     So we could just echo the instructions that users need to access the console.  Maybe that's the path of least resistence.

Reply:
I thought we already did that, but yes maybe we should just tell them to run `minikube service vault` and move on rather than trying to detect the background process and etc.